src/guides/bioinformatics/filtering-and-subsampling.rst

```diff
@@ -8,35 +8,186 @@ sample data.
 .. contents:: Table of Contents
    :local:
 
-Filtering
-=========
+Overview
+========
+
+``augur filter`` provides the flexibility to choose different subsets of input
+data for various types of analysis. There are several options which can be
+categorized based on the information source and selection method.
+
+Information source:
+
+- **Metadata-based** options work with information available from
+  ``--metadata``.
+- **Sequence-based** options work with information available from
+  ``--sequences`` or ``--sequence-index``.
+
+Selection method:
+
+- **Preliminary** options work by selecting or dropping sequences that match
+  certain criteria.
+- **Subsampling** options work by selecting sequences using rules based on final
+  output size. These are applied after all preliminary options and before any
+  force-inclusive options.
+- **Force-inclusive** options work by ensuring sequences that match certain
+  criteria are always included in the output, ignoring all other filter options.
+
+.. list-table:: Categories for filter options
+   :header-rows: 1
+   :stub-columns: 1
+
+   * -
+     - Metadata-based
+     - Sequence-based
+   * - Preliminary
+     - * ``--min-date``
+       * ``--max-date``
+       * ``--exclude-ambiguous-dates-by``
+       * ``--exclude``
+       * ``--exclude-where``
+       * ``--query``
+     - * ``--min-length``
+       * ``--max-length``
+       * ``--non-nucleotide``
+
+   * - Subsampling
+     - * ``--subsample-max-sequences``
+       * ``--group-by``
+       * ``--sequences-per-group``
+       * ``--group-by``
```

Comment:
Typo - `group-by` is included twice

Reply:
Fixed: https://github.com/nextstrain/docs.nextstrain.org/blob/96f68da125c4c34d05b5f5fbf41c22bc4b399179/src/guides/bioinformatics/filtering-and-subsampling.rst#L53-L59